pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -43,6 +43,20 @@ var (
 		"packetDeltaCount",
 		"octetDeltaCount",
 	}
+	IANAInfoElementsIPv6 = []string{
```

Comment:
Don't we need AntreaInfoElementsIPv6 for destinationClusterIPv6, when clusterIP/svcCIDR is v6? Pod-To-Service flow with IPv6 will not have the clusterIP. 

As a side note, we probably need to improve unit test TestFlowExporter_sendDataRecord by considering values in the input flow to cover scenarios with IPv4 and IPv6 to catch these errors. Should be considered for future.

Reply:
Thanks for reminding me. Updated.
Updated tests as well.